_posts/news/2018-02-01-announcing-scalajs-1.0.0-M3.md

```diff
@@ -0,0 +1,463 @@
+
+* Scala 2.10.x is not supported anymore, nor building on JDK 6 and 7.
+* The Tools API has been split in 3 separate artifacts.
+
+<!--more-->
+
+Please report any issues [on GitHub](https://github.com/scala-js/scala-js/issues).
+
+## Preparations before upgrading from 0.6.x
+
+Before upgrading to 1.0.0-M3, **we strongly recommend that you upgrade to Scala.js 0.6.20 or later**, and address all deprecation warnings.
+Since Scala.js 1.0.0-M3 removes support for all the deprecated features in 0.6.x, it is easier to see the deprecation messages guiding you to the proper replacements.
+
+Additionally to the explicitly deprecated things, make sure to use `scalaJSLinkerConfig` instead of the following sbt settings:
+
+* `scalaJSSemantics`
+* `scalaJSModuleKind`
+* `scalaJSOutputMode`
+* `emitSourceMaps`
+* `relativeSourceMaps`
+* `scalaJSOptimizerOptions`
+
+## Upgrade to 1.0.0-M3 from 0.6.20 or later
+
+As a first approximation, all you need to do is to update the version number in `project/plugins.sbt`:
+
+{% highlight scala %}
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.0.0-M3")
+{% endhighlight %}
+
+In addition, if you use some of the components that have been moved to separate repositories, you will need to add some more dependencies in `project/plugins.sbt`:
+
+If you use `crossProject`:
+
+* Add `addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "0.3.1")` in `project/plugins.sbt`
+
+If you use `jsDependencies` (or rely on the `jsDependencies` of your transitive dependencies):
+
+* Add `addSbtPlugin("org.scala-js" % "sbt-jsdependencies" % "1.0.0-M3")` in `project/plugins.sbt`
+* Add `.enablePlugins(JSDependenciesPlugin)` to Scala.js `project`s
+* Add `.jsConfigure(_.enablePlugins(JSDependenciesPlugin))` to `crossProject`s
+
+If you use the Node.js with jsdom environment:
+
+* Add `libraryDependencies += "org.scala-js" %% "scalajs-env-nodejs" % "1.0.0-M3"` in `project/plugins.sbt`
+
+If you use the PhantomJS environment:
+
+* Add `addSbtPlugin("org.scala-js" % "sbt-scalajs-env-phantomjs" % "1.0.0-M3")` in `project/plugins.sbt`
+
+Finally, if your build has
+
+{% highlight scala %}
+scalacOptions += "-P:scalajs:sjsDefinedByDefault"
+{% endhighlight %}
+
+you will need to remove it (Scala.js 1.x always behaves as if `sjsDefinedByDefault` were present).
+
+This should get your build up to speed to Scala.js 1.0.0-M3.
+From there, you should be able to test whether things go smoothly, or whether you are affected by the breaking changes detailed below.
+
+## Breaking changes
+
+This section discusses the backward incompatible changes, which might affect your project.
+
+### Scala 2.10.x is not supported anymore, nor building on JDK 6 and 7
+
+The title says it all: you cannot use Scala.js with
+
+{% highlight scala %}
+scalaVersion := "2.10.x" // for any x
+{% endhighlight %}
+
+anymore.
+Note that you can still use the sbt plugin with sbt 0.13.16+, although it runs itself on 2.10.x.
```

Comment:
s/although/even though/? "Although" feels like it introduces a caveat, not a clarification.